Libplanet.Extensions.Cocona/Commands/Key/DerivationCommand.cs

```diff
@@ -0,0 +1,34 @@
+namespace Libplanet.Extensions.Cocona.Commands.Key
+{
+    using System;
+    using global::Cocona;
+    using Libplanet.Crypto;
+
+    public class DerivationCommand
+    {
+        public void PrivateKey(
+            [Argument("PRIVATE-KEY")]
+            string privateKeyHex,
+            [Option]
+            bool publicKey = false,
+            [Option]
+            bool address = false)
+        {
+            if (!(publicKey ^ address))
+            {
+                throw new CommandExitedException($"Only one flag should be used between {nameof(publicKey)} and {nameof(address)}", -1);
+            }
+
+            var privateKey = new PrivateKey(ByteUtil.ParseHex(privateKeyHex));
+            if (address)
+            {
+                Console.Out.WriteLine(privateKey.ToAddress().ToHex());
```

Comment:
There's convention that addresses are prefixed with `0x`, hence let's follow that.  And that's what `Address.ToString()` does:

https://github.com/planetarium/libplanet/blob/0b775f27de4890938930cc75fcb4716d0c581dec/Libplanet/Address.cs#L222-L225

```suggestion
                Console.Out.WriteLine(privateKey.ToAddress());
```

FYI there's [`TextWriter.WriteLine(object)` method overload][1] which takes an `object` and writes its `.ToString()`.

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.io.textwriter.writeline?view=netstandard-2.0#System_IO_TextWriter_WriteLine_System_Object_